_src/assets/img/jupyter-logo.svg

```diff
@@ -1 +1,5 @@
+>>>>>>> 2137dcfd320076c41ce610930ae5a1e5d48d5cb8
```

Comment:
@gweidner Had a good find here.  This svg file has conflict text in it, which is why it wasn't working.  Can you just replace this svg with the correct svg?  I think it's either line 2 or 4 (or just replace the file), and then the svg should work again.